onboarding-site/entrypoint.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 PROC=$(cat /proc/cpuinfo | grep "model name" | sed -n -e 's/^model name.*: //p' | head -n1)
 DEVICE_TYPE=$(sed -e 's/^device_type=//' /var/lib/mender/device_type 2>/dev/null || uname -m)
-MENDER_VERSION=$(mender -version 2>/dev/null | head -n1)
+MENDER_VERSION=$( ( mender-update --version 2>/dev/null || mender -version 2>/dev/null ) | head -n1)
```

Comment:
```suggestion
MENDER_VERSION=$({ mender-update --version 2>/dev/null || mender -version 2>/dev/null; } | head -n1)
```
perhaps there is no need for the extra subshell here?

Reply:
Good suggestion. Thank you.